runtimes/nodejs/client-get-signed-url.js

```diff
@@ -25,7 +26,7 @@ async function main(args) {
     response = await cos.getSignedUrl(params.operation, {
       Bucket: params.bucket,
       Key: params.key,
-      Expires: expires,
+      Expires: params.expires,
     });
```

Comment:
should this have a default still?

Reply:
yep, default is set in the getParamsCOS function and is the same as the default from s3.